CHANGELOG.md

```diff
@@ -29,14 +29,15 @@ release.
 
 <table>
 <tr>
-  <th title="Current"><a href="doc/changelogs/CHANGELOG_V20.md">20</a> (Current)</th>
+  <th title="LTS Until 2026-04"><a href="doc/changelogs/CHANGELOG_V20.md">20</a> (LTS)</th>
   <th title="Current"><a href="doc/changelogs/CHANGELOG_V19.md">19</a> (Current)</th>
   <th title="LTS Until 2025-04"><a href="doc/changelogs/CHANGELOG_V18.md">18</a> (LTS)</th>
   <th title="LTS Until 2023-09"><a href="doc/changelogs/CHANGELOG_V16.md">16</a> (LTS)</th>
```

Comment:
Should this be updated now that it's past 2023-09?

Reply:
Possibly, but at this stage I don't want to add more things to this release. Backporting the relevant doc changes from main can be considered for a future Node.js 20 release.